unblob/handlers/compression/bzip2.py

```diff
@@ -81,7 +81,7 @@ def bzip2_recover(self, file: io.BufferedIOBase, start_offset: int) -> int:
                 # This can be negative, but we don't care, because we also count found blocks
```

Comment:
This makes perfect sense to me, I don't know why we added 1 in the first place. Another optimization to get rid of the "this can be negative" message would be:

```python3
if buff == BLOCK_HEADER or buff == BLOCK_ENDMARK:
    blocks_found += 1

if buff == BLOCK_ENDMARK:
    current_block_end = bits_read - COMPRESSED_MAGIC_LENGTH
```

Reply:
In reality it no longer can be negative, because we have just found a complete block.